cookbook/controller/service.rst

```diff
@@ -76,6 +76,13 @@ Then you can define it as a service as follows:
             'AppBundle\Controller\HelloController'
         ));
 
+.. caution::
+
+    If you are managing your routes with PHP annotation, you may need to add an
```

Comment:
It is not a ``may``. It is always needed (otherwise the controller is not used as a service)